### DISCUSSION TITLE
is there any way to load extension in pure python mode

### DISCUSSION THREAD

**Discussion by drinktea690115** [seleniumbase/SeleniumBase#1791] [Q&A] [answered]
as title
is there any way to load extension with pure python
such as the code below

from seleniumbase import SB
with SB() as sb:
    sb.open("https://www.google.com.tw")

thanks for help

**Comment by drinktea690115** [comment]
i have tried the code below
import time
from seleniumbase import SB

    with SB(browser="chrome",uc=True,extension_zip="D:\\abc.crx") as sb:
        sb.open("https://accounts.google.com")
        print('ok')
        time.sleep(50000)

but not work
and the code below works,so if it possible for the seleniumbase to use the extension_zip parameter?
    with SB(browser="chrome",uc=True,extension_dir="D:\\abc") as sb:
        sb.open("https://accounts.google.com")
        print('ok')
        time.sleep(50000)

**Comment by mdmintz** [comment, marked as answer]
When using UC Mode, you must use `extension_dir` (instead of `extension_zip`) when loading Chrome extensions.
